userspace/falco/app/actions/configure_interesting_sets.cpp

```diff
@@ -177,8 +167,7 @@ falco::app::run_result falco::app::actions::configure_interesting_sets(falco::ap
 	}
 
 	s.selected_sc_set.clear();
-	s.selected_tp_set.clear();
-	
+
```

Comment:
We don't have a `tracepoint` API anymore. We use PPM_SC for everything.
Tracepoints are already extracted by the ruleset, just like syscalls.